eventhubs/data-plane/azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/AzureBlobLease.java

```diff
@@ -97,4 +99,26 @@ String getStateDebug() {
         }
         return retval;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) { return false; }
```

Comment:
nit: new line after `{`. I think there would be a checkstyle error about this too.

Reply:
Will correct it. Curious why CheckStyle has no warning for the error.

Reply:
the CheckStyle has produced warning for the error. Corrected it.